CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+Contribute to Scrap Packed Explorer
+=====================
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow][githubflow], So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow][githubflow]). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Issue that pull request!
+
+## Any contributions you make will be under the MIT Software License
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues](https://github.com/romibi/Scrap-Packed-Explorer/issues)
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/romibi/Scrap-Packed-Explorer/issues/new); it's that easy!
+
+**Great Bug Reports** tend to have:
+
+- A quick summary and/or background
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can
+- What you expected would happen
+- What actually happens
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+People *love* thorough bug reports. I'm not even kidding.
+
+## Use a Consistent Coding Style
+
+### Naming Conventions
+* Every class and method should be named using **PascalCase** convention (example: `void ReadFile()`)
+* Every method argument should be named using **camelCase** convetion with prefix _p__ (example: `string p_filePath`)
+* Every variable should be named using **camelCase** convention (example: `int maxLength`)
+* Every application's command line argument should be named using **camelCase** convention (example: `--keepBackup`)
+<!-- TODO: change args to kebab-case? (--keep-backup) -->
+
+### Brackets and indentation
+Indentation must be done with 4 spaces. Lines should be ended with `\r\n` (Windows style)
+
+Brackets should **always** start on a new line. If brackets **can** be removed - they **should** be removed.
+
+`else`, `catch` and `finally` blocks should be on next line after closing bracket
+For control-flow blocks (`if`, `while`, `for`, etc.) the brackets containing the condition should be separated from keyword with a space.
```

Comment:
This is still open to debate for me …
If the .editorconfig can be set up like that and if we update all current code to that …
i'd prefer
```c#
try {
    if (a == b) {
        // Some code...
        // Some code...
        // Some code...
    } else
        // One line of code
} catch {
    // Error handling code
}
```

Reply:
It will not delete unnecessary brackets.
Maybe there is a way to configure it like this but I didn't foud one

Reply:
But if configure that all brackets are necessery it will add them

Reply:
Oh, just noticed that opening brackets are on the same line with keywords.
Yes, I belive we can do this

Reply:
And what about classes ans functions? Do we open them on next line?

Reply:
I'd prefer not … let's not have opening brakets alone on a line …
I'd say that's wasteful usage of vertical height …
The only place i can think of where i'm ok with that … would be for namespaces (as that is something you'd nearly always only once per file) … but for consistency sake even there i'd prefer it at the end of the line …

Reply:
you have updated the example but not the text above